Example/ViewController+MiniAppMessageProtocol.swift

```diff
@@ -25,6 +26,17 @@ extension ViewController: MiniAppMessageProtocol, CLLocationManagerDelegate {
         }
     }
 
+    func requestCustomPermissions(permissions: [MASDKCustomPermissionModel], completionHandler: @escaping (Result<[MASDKCustomPermissionModel], Error>) -> Void) {
+        if let viewController = UIStoryboard(name: "Main", bundle: nil).instantiateViewController(withIdentifier: "CustomPermissionsTableViewController") as? CustomPermissionsTableViewController {
```

Comment:
I think this should first check if all of the permissions are already granted. If they are already granted then it should immediately invoke the callback to the JS bridge. This is because we will probably suggest that the Mini Apps call `requestCustomPermission` every time they use any of the user data features (because the user could revoke the permissions at any time from the Host App settings). What do you think?

Reply:
We are already checking in the SDK if the user has already granted permissions (before requesting the host app) and if yes, then we are sending the cached response back to the Mini app. [Check here](https://github.com/rakutentech/ios-miniapp/pull/129/files#diff-ee09dac8c84e5da89672c0c27e4f6401R37)